kratos/utilities/variable_utils.h

```diff
@@ -384,11 +384,33 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
         KRATOS_CATCH("")
     }
 
+
+    /**
+     * @brief Clears the container data value container
+     * @param rContainer Reference to the objective container
+     */
+    template< class TContainerType>
+    void ClearNonHistoricalData(TContainerType& rContainer)
+    {
+        KRATOS_TRY
+
+        const auto it_cont_begin = rContainer.begin();
+
+        #pragma omp parallel for
+        for (int k = 0; k< static_cast<int> (rContainer.size()); ++k) {
+            auto it_cont = it_cont_begin + k;
+            auto& r_data = it_cont->Data();
```

Comment:
I don't think you need this reference, do you?
```suggestion
            it_cont->Data().Clear();
```

Reply:
If I apply this suggestion the next line will stay the sanme

Reply:
I know :)